src/coreComponents/unitTests/xmlTests/testXMLFile.cpp

```diff
@@ -35,6 +41,197 @@ TEST( testXML, testXMLFile )
   EXPECT_TRUE( problemManager.getFieldSpecificationManager().hasGroup< FieldSpecificationBase >( "v0" ) );
 }
 
+/**
+ * @brief Returns the xml nodes and attribute in hierarchy that GEOS is supposed to read in
+ * the Group hierarchy.
+ * @param document root xml document (which potentially contains includes).
+ * @param targetNode the function will be called on this node and its children.
+ * @param elementNames the list of xml elements names with the form "GroupName" or
+ * "GroupName/WrapperName". The node name is supposed to be the name attribute value, it it exists,
+ * or the tag name.
+ */
+void getGEOSElementsRecursive( xmlDocument const & document, xmlNode const & targetNode,
```

Comment:
I would suggest getElementsRecursive just to minimize search&replace in case of a new name change